Implement elections list

selankon opened the pull request.

selankon committed:
Refactor paths names

selankon committed:
Implement elections list

selankon committed:
Fix paths variables

selankon committed:
Add debounceTime to InputSearch

selankon committed:
Implement useQuery params

selankon committed:
Implement process by type

selankon committed:
Use no total count paginator

selankon committed:
Delete comment

selankon committed:
Bump chakra-components

selankon committed:
Bump extended-sdk

selankon committed:
Fix lintern issues

elboletaire commented on src/theme/components/StatusBage.tsx:
```diff
@@ -0,0 +1,25 @@
+    height: '18px',
+    fontStyle: 'normal',
+    fontWeight: '700',
+    fontFamily: 'Manrope',
```
Always define fallback fonts.

selankon replied:
I delete the fontFamily attribute because is inherited from the old explorer and is not actually needed for the moment. On a near future we probably will style the explorer adding custom fonts, so a this moment is not needed to define this... 

d912f96

elboletaire commented on src/theme/components/StatusBage.tsx:
```diff
@@ -0,0 +1,25 @@
+  },
+})
+
+export const tagTheme = defineMultiStyleConfig({
```
It is a bit confusing to have the Tag theme under the StatusBadge theme file (btw there's a typo on the filename, I've just noticed.. it's missing a `d`).

I'd name the file `Tag.tsx` and also the `tagTheme` would be `Tag` (this way matches with the file name and also can be imported easily, since it's the name used by the component in the theme).

selankon replied:
Absolutely agree. As before, this code is copy pasted from the old explorer. I fix it: 0adccc2

selankon committed:
Delete font family

I delete the fontFamily attribute because is inherited from the old explorer and is not actually needed for the moment. On a near future we probably will style the explorer adding custom fonts, so a this moment is not needed to define this...

selankon committed:
Refactor file name
